### PR TITLE
docs: add project k8s

### PR DESCRIPTION
**Pull request by mildronize**
Please review my PR :)

### CODE REVIEW

File: content/projects/kubernetes/kubernetes-manifest-generator.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Original Author: [Sirinat Paphatsirinatthi](https://github.com/dmakeroam)
+Source Code: https://github.com/kubeopsskills/levis
+
+The aim of this project is to simplify Kubernetes configuration.
+You just write a few line, then deployed to Kubernetes cluster
+
+**Levis** is Kubernetes Manifest Generator for simply creating Kubernetes applications. Levis generates pure Kubernetes YAML - you can use Levis to define Kubernetes applications with YAML syntax for any Kubernetes clusters running anywhere.
+
+This is an project built with ❤️ by [KubeOps Skills](https://www.kubeops.guru).
```
Comment (aaronamm) [inline content/projects/kubernetes/kubernetes-manifest-generator.md:20]: @mildronize @dmakeroam I like this.

File: content/projects/kubernetes/kubernetes-manifest-generator.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+categories:
+  - project
+tags:
+  - kubernetes
+---
+
+Original Author: [Sirinat Paphatsirinatthi](https://github.com/dmakeroam)
+Source Code: https://github.com/kubeopsskills/levis
+
+The aim of this project is to simplify Kubernetes configuration.
```
Comment (dmakeroam) [inline content/projects/kubernetes/kubernetes-manifest-generator.md:15]: The aim of this project is to simplify Kubernetes configuration.
You just write a few lines, then deploy the configuration to Kubernetes cluster

**Comment by mildronize** [comment]
@aaronamm @dmakeroam I've done.

**Comment by dmakeroam** [comment]
LGTM

**Review by dmakeroam — APPROVED** [review]

**Comment by mildronize** [comment]
@dmakeroam Thanks a lot, I'll merge.

**Pull request merged**